### DISCUSSION TITLE
manual http-api update & notification for new images without auto-updating

### DISCUSSION THREAD

**Discussion by killtux** [containrrr/watchtower#1334] [Q&A] [answered]
hello!

i am not sure what flags to set. i am currently using http-api-update but like to get an notification if there is a new image available so i can trigger the http api request.

do i have to combine it with --http-api-periodic-polls or how to prevent scheduled updates and only get notfications but keep the ability to force an update over http-api?

**Comment by piksel** [comment, marked as answer]
That actually is not possible to do right now. The problem is that both the `http-api`-triggered updates, and the scheduled updates are using the same configuration. So if you put the `--monitor-only` flag to only get notifications, it will apply for the http api as well.

You could launch a separate `watchtower --run-once` to do the update instead though, but you would need to create your own wrapper for it if you want to trigger it via an API of course.

**Reply by killtux** [reply]
thanks for clarifying. wouldnt this be an use case to get notified if there is an update available and then, if you think its the right moment start the update over api so you dont have to login and do everything manually?

usecase for me: i have to keep some dockers up to date to dont run into problems but also had some issues in past if the automatic update ran in the night and failed and i couldnt troubleshoot anything the next day because i didnt had time but also devices stopped working and this was also a big issue. so i'd like to get some notifications if there are new versions available and then hit the remote update button to start the update when i have some time window  to troubleshoot and fix if anything wents wrong.
